macOSUpgrade.sh

```diff
@@ -368,6 +400,24 @@ cleanExit() {
     exit "$1"
 }
 
+isSystemRequirementErrors() {
```

Comment:
In view of the existing code, you might want to prefix this function name with `validate`.

Reply:
Do you mean the function name is better liker "validate_system_requirement _error" than "isSystemRequirementError"? 

I think the  primary work of this function is not 'validate' but show dialogue when those results of validate has error. 

So changing the name to like "validate_system_requirement _error", I think that is not what this function is actually doing.

Reply:
I think it was a validate because it was doing `cleanExit` inside the function.
It may be common for `isXXX` to return true or false as a value.

Reply:
Humm,,,　`cleanExit()` is not for validating function, isn't it?

> It may be common for isXXX to return true or false as a value.
How about `check_system_requirement()` instead of that?